src/openms/include/OpenMS/ANALYSIS/OPENSWATH/MRMTransitionGroupPicker.h

```diff
@@ -498,6 +502,28 @@ namespace OpenMS
     void findLargestPeak(std::vector<MSChromatogram >& picked_chroms, int& chr_idx, int& peak_idx);
     
     /**
+      @brief Will use the chromatogram to estimate the background noise 
+        using the average intensity to the left and right of the peak borders,
+        and then subtract it
+
+        The background is estimated by averaging the noise on either side of the
+        peak and then subtracting that from the total intensity.
+    */
+    void calculateBgEstimationAverage_(const MSChromatogram& chromatogram,
+                                  double best_left, double best_right, double & background, double & avg_noise_level);
+    
+    /**
+      @brief Will use the chromatogram to estimate the background noise 
+        using the exact intensity of the left and right peak borders,
+        and then subtract it
```

Comment:
it seems like the description of the two functions is now exactly the same, but they dont do the same. can we rectify that, I would suggest you update yours